openedx/frontend-app-course-authoring
# feat: add component preview to library authoring [FC-0062]

## Description
This PR adds an XBlock preview to the library authoring sidebar.
![image](https://github.com/user-attachments/assets/2eded04f-fd6c-4483-a77d-b7b3cbe20156)

## More information
Part of:
 - https://github.com/openedx/frontend-app-course-authoring/issues/1189
 
Depends on:
- https://github.com/openedx/edx-platform/pull/35473

## Testing Instructions
- Checkout this branch and https://github.com/openedx/edx-platform/pull/35473
- Open the library authoring home for a library with some components
- Click on the component and check the preview on the sidebar. You can't interact with the XBlock here.
- Clicking on expand should open a modal dialog with the component preview. In this preview, you can interact with the XBlock
___
Private ref: [FAL-3801](https://tasks.opencraft.com/browse/FAL-3801)
